src/box/errcode.h

```diff
@@ -431,6 +431,7 @@ struct errcode_record {
 	 */ \
 	_(ER_READ_VIEW_BUSY, 285,		"The read view is busy") \
 	_(ER_READ_VIEW_CLOSED, 286,		"The read view is closed") \
+	_(ER_INVALID_VCLOCK, 287,		"Invalid vclock") \
```

Comment:
We should add `"value"` as payload.

Reply:
Done. Note that it's not tested - now we never raise this error, only log it, and `diag_log` doesn't log payload for now.